Makefile

```diff
@@ -27,20 +27,14 @@ test-ui:
 	$(UIDOCKER) npm run test
 
 lint:
-	flake8 aleph/
+	ruff check .
 
 lint-ui:
 	$(UIDOCKER) npm run lint
 
-format:
-	black aleph/
-
```

Comment:
Hm, but black doesn’t replace a formatter, right?

> As a project, Ruff is designed to be used alongside Black and, as such, will defer implementing stylistic lint rules that are obviated by autoformatting.

https://beta.ruff.rs/docs/faq/#is-ruff-compatible-with-black

Reply:
You are totally right, my bad. Reverting this.